framework/console/controllers/BaseMigrateController.php

```diff
@@ -124,14 +124,19 @@ public function actionUp($limit = 0)
         }
         $this->stdout("\n");
 
+        $applied = 0;
         if ($this->confirm('Apply the above ' . ($n === 1 ? 'migration' : 'migrations') . '?')) {
             foreach ($migrations as $migration) {
                 if (!$this->migrateUp($migration)) {
+                    $this->stdout("\n$applied from $n " . ($applied === 1 ? 'migration was' : 'migrations were') ." applied.\n", Console::FG_RED);
```

Comment:
`migration` is still common part so only `was` and `were` could be in condition.